references/cicd-pipeline/ci-config/jenkins/Jenkinsfile

```diff
@@ -39,6 +39,7 @@ pipeline {
         stage('Install dependencies') {
           steps {
             sh "npm install --silent --no-fund"
+            echo sh(script: 'env|sort', returnStdout: true)
```

Comment:
Can you elaborate why you think this is required? 
The pipeline appears to run fine on my end and the nightly. Also I'd rather not print the full env to STDOUT.